src/main/java/uk/gov/hmcts/sptribs/caseworker/service/HearingService.java

```diff
@@ -118,10 +118,24 @@ public void updateHearingList(CaseData caseData) {
         for (ListValue<Listing> listingListValue : caseData.getHearingList()) {
             String hearingName = caseData.getCicCase().getHearingList().getValue().getLabel();
             if (hearingName.contains(listingListValue.getValue().getHearingTime())
```

Comment:
Since you have the same long boolean expression to evaluate, it be worth moving it to its own method
```suggestion
            if (hearingNameContainsListValue(hearingName, listingListValue) {
                listingListValue.setValue(caseData.getListing());
                break;
            }
            
            private boolean hearingNameContainsListValue(String hearingName, ListValue<Listing> listingListValue) {
                return hearingName.contains(listingListValue.getValue().getHearingTime())
                             && hearingName.contains(listingListValue.getValue().getHearingType().getLabel())
                             && hearingName.contains(listingListValue.getValue().getDate().format(dateFormatter));
            }
```

Reply:
Don't click commit suggestion, I just wanted it to format in a clear way

Reply:
refactored as suggested